apps/ex_wire/lib/ex_wire/sync.ex

```diff
@@ -18,9 +18,21 @@ defmodule ExWire.Sync do
   alias Block.Header
   alias Blockchain.Block
   alias ExWire.Config
-  alias ExWire.Packet.{BlockBodies, BlockHeaders, GetBlockBodies, GetBlockHeaders}
+
+  alias ExWire.Packet.{
```

Comment:
In elixir can you alias all of a module? i.e. `alias ExWire.Packet._`?

Reply:
Not sure, but I like keeping the imports kind of explicit here. It basically is saying `sync.ex` has gotten too complicated :)